pkgs/tools/text/mecab/ipadic.nix

```diff
@@ -1,18 +1,19 @@
 { stdenv, fetchurl, mecab-nodic }:
 
-stdenv.mkDerivation {
```

Comment:
what do you think of using the `stdenv.mkDerivation (finalAttrs: {` instead ? it avoids "rec" and is powerful

Reply:
It's kinda ugly. Done.